pkg/kubelet/rkt/rkt.go

```diff
@@ -1785,6 +1785,10 @@ func (r *Runtime) GetNetNS(containerID kubecontainer.ContainerID) (string, error
 	return netnsPathFromName(makePodNetnsName(kubetypes.UID(containerID.ID))), nil
 }
 
+func (r *Runtime) GetPodContainerID(pod *kubecontainer.Pod) (kubecontainer.ContainerID, error) {
+	return kubecontainer.ContainerID{ID: string(pod.ID)}, nil
```

Comment:
I let this sneak through, we are bit by this again. The returning container ID is using the rkt pod's uuid, not the pod's UID. @dcbw